Docs/creating an aggregate (#480)
leancodepl/corelibrary: docs/README.md

 3. [Benchmarks](./benchmarks/README.md).
 4. [Domain](./domain/README.md).
 5. [Architecture decision records](./adrs/README.md).

## Domain Driven Design

> LeanCode Core Library is strongly based on concepts of Domain Driven Design. If you are not familiar with this approach to developing software please check out the following books:

Reviewer: ```suggestion
LeanCode Core Library is strongly based on concepts of Domain Driven Design. If you are not familiar with this approach to developing software, you can check these books:
```

less aggressive ;P
